proj/src/main/webapp/chat.js

```diff
@@ -734,13 +738,72 @@ function addUserInfoToDom(userObj) {
     }
     profile.querySelector("#user-bio").innerText = userObj.bio;
 
-    const tagList = profile.querySelector("#user-tags");
-    tagList.innerHTML = '';
     for (tag in userObj.tags) {
         if (userObj.tags.hasOwnProperty(tag)) {
-            const tagNode = document.createElement('li');
-            tagNode.innerText = tag;
-            tagList.appendChild(tagNode);
+            addTag(tag, userObj.uid);
         }
     }
-}
\ No newline at end of file
+}
+
+// adds tag input
+function addTagsToDom(uid) {
+    tagsRef = firebase.database().ref('/users/'+uid+'/allTags');
+    tagContainer = document.querySelector('.tag-container');
+
+    const tagInput = document.createElement('input');
+    tagInput.id = "tag-input";
+    tagContainer.append(tagInput);
+    tagInput.addEventListener('keyup', function(event) {
+        if (event.keyCode === 13) {
```

Comment:
can you add a comoment saying what key is 13? I think it is enter but I'm not sure.

Reply:
fixed